lib/tasks/server/livereload-server.js

```diff
@@ -40,7 +40,7 @@ module.exports = Task.extend({
   start: function(options) {
     var tlroptions = {};
     tlroptions.ssl = options.ssl || false;
-    tlroptions.host = '0.0.0.0';
+    tlroptions.host = options.liveReloadHost || '0.0.0.0';
```

Comment:
```
   { name: 'live-reload-host',  type: String, description: 'Defaults to host', aliases: ['lrh'] },
     { name: 'live-reload-port', type: Number, description: '(Defaults to port number + 31529)', aliases: ['lrp']},
```

don't appear to have defaults, instead the defaults exist: in the `start` function of livereload-server.js

Seems like they might be better suited as defaults in the command ?